static/data/statsvar_path.json

```diff
@@ -8679,13 +8679,27 @@
   6,
   2
  ],
+ "Count_Person_3OrMoreYears_Female_EnrolledInPrivateSchool": [
```

Comment:
basically these are all the duplicate nodes so far

Reply:
right